src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,36 @@
+package tests;
+
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.Selenide;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+
+import static core.enums.PropertyNames.BROWSER;
+import static core.enums.PropertyNames.BROWSER_SIZE;
+import static core.enums.PropertyNames.DRIVER_MANAGER_ENABLED;
+import static core.enums.PropertyNames.HEADLESS;
+import static core.enums.PropertyNames.WEB_DRIVER_LOGS_ENABLED;
+import static core.utils.services.ConfigurationService.getProperty;
+import static java.lang.Boolean.parseBoolean;
+
+abstract public class BaseTest {
+
+  @BeforeEach
```

Comment:
Are you sure you need to set up properties before each test? I assume you can do it once using @BeforeAll annotation

Reply:
Done